src/components/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+import "./ErrorMessage.sass";
+
+const ErrorMessage = ({ errorMessage }) =>
+  errorMessage && <h1 className="error-message">{errorMessage}</h1>;
```

Comment:
I would probably just render error message here, instead of doing a conditional.
And leave the condition in consumer's logic.

Leaving this that way could lead to assumption that 
`<ErrorMessage />` will be always shown.

Reply:
Btw are you sure it should be h1?